mocks/inference-service/inference_service.py

```diff
@@ -15,25 +15,45 @@
 """Mock server that can be used instead of fully functional Inference Service."""
 
 
-from fastapi import FastAPI
+from fastapi import Request, FastAPI
 
 
 app = FastAPI()
 
 
 @app.get("/upgrade-risks-prediction")
-def upgrade_risk_prediction_mock():
+async def upgrade_risk_prediction_mock(request: Request):
     """Request handler for REST API endpoint to return upgrade prediction prediction."""
-    return {
+    data = await request.json()
+    expected_data = {
+        "alerts": [
+            {
+                "name": "SomeCriticalAlert",
+                "namespace": "openshift-kube-apiserver",
+                "severity": "critical",
+            }
+        ],
+        "operator_conditions": [
+            {"name": "authentication", "condition": "Degraded", "reason": "AsExpected"}
+        ],
+    }
+
+    result = {
         "upgrade_recommended": False,
-        "upgrade_risks_predictors": {
-            "alerts": [],
-            "operator_conditions": [
-                {
-                    "name": "authentication",
-                    "condition": "Degraded",
-                    "reason": "AsExpected"
-                }
-            ]
-        }
+        "upgrade_risks_predictors": {"alerts": [], "operator_conditions": []},
     }
+    if data == expected_data:
+        result = {
+            "upgrade_recommended": True,
```

Comment:
I think that @juandspy already get rid of the "upgrade_recommended" field in the response, maybe this schema is outdated?

Reply:
According to [this](https://gitlab.cee.redhat.com/ccx/ccx-upgrades-data-eng/-/blob/main/ccx_upgrades_data_eng/models.py#L80), the field should still be there.